### PR TITLE
REPO-5072: Update version of acs-packaging to 7.0

### PR DESCRIPTION
**Pull request by aforascu**
- update version to 7.0-SNAPSHOT
- comment out googledrive-repo-community amp since it is not compatible with the new acs-community version
- changes required on amps will be done in https://issues.alfresco.com/jira/browse/REPO-5086

### CODE REVIEW

**Comment by aforascu** [comment]
@montgolfiere Yes, the issue is mostly related to the repo.max.version 6.99 in module.properties of every single amp and I've opened https://issues.alfresco.com/jira/browse/REPO-5086 for fixing this, but there may be other issues encountered (as for ai-renditions - ATS-686). 
I'm not very happy either with the process of commenting out, but I'm not sure that it would be proper to fix this on alfresco-repository and change functionality that is in place and works for a long time now. If I would change the message to an INFO then it may be overlooked and still 7.0 is way ahead - maybe 1 year?

**Comment by montgolfiere** [comment]
I'd say "7.0" was a good time to change this behaviour :-) 

Also, I'm sure we will be regularly releasing early builds. Let's not create unnecessary tech debt from the outset.

**Comment by montgolfiere** [comment]
FYI: GoogleDocs 3.2.0-RC3 no longer has a max version so does not need to be commented-out. 

However, my [recommendation above|https://github.com/Alfresco/acs-community-packaging/pull/706#pullrequestreview-386198576] still applies for all other modules that may be impacted.

File: docker-alfresco/pom.xml [hunk 1]
```diff
@@ -200,14 +200,14 @@
...
                                 </artifactItem>
-                                <artifactItem>
+<!--                                 <artifactItem>
                                     <groupId>org.alfresco.integrations</groupId>
                                     <artifactId>alfresco-googledrive-repo-community</artifactId>
                                     <version>${alfresco.googledrive.version}</version>
                                     <type>amp</type>
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
-                                </artifactItem>
```
Comment (montgolfiere) [inline docker-alfresco/pom.xml:210]: FYI: [GoogleDocs 3.2.0-RC3|https://github.com/Alfresco/acs-community-packaging/blob/develop/pom.xml#L99] no longer has a max version so there should be no need to comment-out.

**Comment by DenisGabriela** [comment]
ACS Community Packaging won't be updated to 7.0 (for now at least), see https://issues.alfresco.com/jira/browse/REPO-5072?focusedCommentId=768630&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-768630. 
Closing this draft.

**Pull request closed without merging**